src/DecFP.jl

```
@@ -177,10 +177,19 @@ for w in (32,64,128)
         end
     end
 
+    i′ = string("Int", Sys.WORD_SIZE)
+    Ti′ = eval(Symbol(i′))
+    @eval begin
+        Base.trunc(::Type{Integer}, x::$BID) = xchk(ccall(($(bidsym(w,"to_",lowercase(i′),"_xint")), libbid), $Ti′, ($BID,), x), InexactError, INVALID | OVERFLOW)
+        Base.floor(::Type{Integer}, x::$BID) = xchk(ccall(($(bidsym(w,"to_",lowercase(i′),"_xfloor")), libbid), $Ti′, ($BID,), x), InexactError, INVALID | OVERFLOW)
+        Base.ceil(::Type{Integer}, x::$BID) = xchk(ccall(($(bidsym(w,"to_",lowercase(i′),"_xceil")), libbid), $Ti′, ($BID,), x), InexactError, INVALID | OVERFLOW)
+        Base.convert(::Type{Integer}, x::$BID) = xchk(ccall(($(bidsym(w,"to_",lowercase(i′),"_xfloor")), libbid), $Ti′, ($BID,), x), InexactError)
```

Some of these don't seem right.  e.g. `convert` should throw an error for non-integer `x`, so simply calling `xfloor` is not correct.

Also, does `xint` have the same rounding behavior as `trunc`?  e.g. `trunc(Int,0.7) === trunc(Int, -0.7) === 0`